cli.go

```diff
@@ -88,55 +91,97 @@ func (cli *CLI) Run(args []string) int {
 		return ExitCodeError
 	}
 
-	// Main function
-	// If disabled test mode, generates real loader
-	if !cli.testMode {
-		cli.loader = loader.NewLoader(c.Debug)
-	}
-	cli.loader.LoadState()
-	cli.loader.LoadTFVars(c.Varfile)
-	if len(args) > 1 {
-		err = cli.loader.LoadTemplate(args[1])
-	} else {
-		err = cli.loader.LoadAllTemplate(".")
-	}
-	if err != nil {
-		fmt.Fprintln(cli.errStream, err)
-		return ExitCodeError
-	}
+	root := "./"
+	dirsToCheck := []string{root}
+	rootInfo, err := os.Stat(root)
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to err
* [staticcheck] this value of err is never used (SA4006)

<!-- Comment by [SideCI](https://sideci.com) -->